rhel/vex/parser_test.go

```diff
@@ -292,6 +294,53 @@ func TestEscapeCPE(t *testing.T) {
 	}
 }
 
+func TestRHTAGParse(t *testing.T) {
```

Comment:
Should this be in `pkg/rhctag/version_test.go` instead?

Reply:
Moved and reformatted to fit in with the current tests there